packages/jaeger-ui/src/components/DeepDependencies/Header/LayoutSettings/index.css

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright (c) 2019 Uber Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.Ddg--LayoutSettings {
+  margin-right: 3em;
+}
+.Ddg--LayoutSettings--optionsTable > tbody > tr {
```

Comment:
these css selectors seem really tied to the html structure... but i suppose the `tbody > tr > td` pattern should be present in all tables...